src/hotspot/share/gc/shenandoah/shenandoahConcurrentMark.cpp

```diff
@@ -243,6 +241,8 @@ void ShenandoahConcurrentMark::concurrent_mark() {
   ShenandoahSATBMarkQueueSet& qset = ShenandoahBarrierSet::satb_mark_queue_set();
   ShenandoahFlushSATBHandshakeClosure flush_satb(qset);
   for (uint flushes = 0; flushes < ShenandoahMaxSATBBufferFlushes; flushes++) {
+    // TODO: ysr : hoist this switch out of the loop & refactor ?? How may
```

Comment:
The default value of ShenandoahMaxSATBBufferFlushes is 5.  I think the number of iterations is typically low.  Given low number of typical iterations, may not be worth it to restructure the loop control.

Reply:
Good point, I'll clear this comment which was inadvertently left behind here.

Reply:
Done.